CCatHome_Regional_Host_Guide.md

```diff
@@ -0,0 +1,228 @@
+
+- Connect with your co-host/CoC facilitator and schedule a meeting with them to test the Zoom and explore the options you both will have available in Zoom
+
+- Connect with the session leads for which you will be hosting the sessions with the session details including the agenda, time in their time zone, zoom link to the call, Etherpad link for the shared notes and [Zoom Instructions](https://docs.google.com/document/d/1_7vxMtcqdn8s-zYOG6THnxZvs4uYmfJUySjNVeVFj98/edit?usp=sharing)
+
+- Invite session leads and co-host to join the session 10 minutes before the start of their session so that they could test their setup, webcam, and microphone
+
+- Double-check the schedule and arrive at the call minimum 15 minutes before the event
+
+#### At the start of the session
+
+- Assign the CoC facilitator and Session lead as Co-Hosts in the Zoom call. Ask session helpers as well if they want to be assigned as Co-Host. (Open the "Participants" tab → find their name → click "More" → select "Make Host" or "Make Co-Host")
+
+- Introduce co-host(s) / CoC facilitator(s) and session lead(s)
+- Welcome the attendees to the call.
+
+- If your session lead has agreed to record the session, tell your attendees that the session will be recorded. Remind them that they can turn off their web cameras if they don't want to be seen in the recordings and that there will be no recording of the breakout discussion.
+
+- Ask your co-host to press record for the video to the cloud. If you do not have a Session Co-Host present, click "Record" at the bottom of the screen and record the session to the cloud.
+
+- If you do not have a Session Co-Host present, help the Session Facilitator manage participants by e.g. muting those who aren't speaking. (Open the "Participants" tab → find their name → click "Mute")
+
+- Convey these reminders:
+
+    - We want to ensure a welcoming and safe space for all our members, and therefore The Carpentries Code of Conduct applies to this session.
+
+    - We have a co-host at this session who is also the CoC facilitator of the call (introduce your co-host), for any report please contact them immediately.
+
+    -  To remind all our attendees, please keep their microphone muted when not speaking.
+
+    - Please use the "raise hand" button on Zoom (click "participants", a window will open on your right side, below the participants list there is a button called "raise hand") or write "/hand" in the chat to indicate to the session lead that they have something to say/add to the discussion
+
+    - Be mindful of others and allow everyone to speak, especially when you have already had a chance to speak up
+
+    - Help the session helpers in taking notes
+
+- Introduce the session lead and hand the session over to them.
+
+- If the Session Facilitator did not want Host privileges, help with any breakout room creation as needed.
+
+#### At the end of the session
+
+- Thank the session leads, helpers, co-host and attendees
+
```

Comment:
There will be a feedback survey - remind participants to fill it out.

Reply:
Added